odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
```

Comment:
Well this gives the wrong result for the "truncated part", otherwise it is all fine. Anyway it is still slightly wrong since by simply right-multiplying with the window we get a problem since we later want to take the derivative along the "u" direction of the detector, but since we have added this window that derivative will get a big jump which causes artefacts. You can inspect that by using the window with `smoothing_width=0`, if you want.

Anyway those artefacts do not go away as we improve the discretization, but persist. In this code we do as everyone else does and smooths the edges (via smoothing_width) which causes that derivative to "level out nicely" and we dont get any major artefacts. The problem with this is that by doing  this smoothing we lose mathematical exactness.

Katsevich solves this by proposing a method that does not need a "smoothing" style hack by doing something else than simply multiplying with a window.

Anyway, we an arbitrarily good discretization this method can get arbitrarily close to being exact, something that for example cannot be said about FDK.